tests/test_ISOSR.py

```diff
@@ -233,6 +274,7 @@ def test_attach_smb_via_xemount_no_version(self, context,
         context.setup_error_codes()
         smbsr = self.create_smbisosr(options='-o username=administrator,password=password')
         smbsr.attach(None)
+        pread.assert_not_called()
```

Comment:
This is new in 3.5 and without `autospec` mock will just invent a new `assert_not_called` method on the mock :(

Need to use `self.assertEqual(0, pread.call_count)`.

If possible add the `autospec=True` attribute to the `mock.patch` attribute as well but that may have other issues, it doesn't always play nice with optional parameters.

Reply:
fixed